src/fireedge/src/modules/containers/Support/Support.js

```diff
@@ -0,0 +1,286 @@
+              tags={props.selectedRows}
+              handleElement={props.handleElement}
+              onDelete={props.handleUnselectRow}
+            />
+          )}
+          info={(props) => {
+            const propsInfo = {
+              ticket: props?.selectedRows?.[0]?.original,
+              selectedRows: props?.selectedRows,
+            }
+            props?.gotoPage && (propsInfo.gotoPage = props.gotoPage)
+            props?.unselect && (propsInfo.unselect = props.unselect)
+
+            return <InfoTabs {...propsInfo} />
+          }}
+        />
+      </TranslateProvider>
+    </MuiProvider>
+  )
+}
+
+/**
+ * Displays details of a support ticket.
+ *
+ * @param {Ticket} ticket - Support ticket to display
+ * @param {Function} [gotoPage] - Function to navigate to a page of a support tickets
+ * @param {Function} [unselect] - Function to unselect a support ticket
+ * @returns {ReactElement} Ticket details
+ */
+const InfoTabs = memo(({ ticket, gotoPage, unselect }) => {
+  const [getComments, { isFetching }] =
+    SupportAPI.useLazyGetTicketCommentsQuery()
+  const id = ticket?.id
+
+  const { settings: { FIREEDGE: fireedge = {} } = {} } = useAuth()
+  const { FULL_SCREEN_INFO } = fireedge
+  const { fullViewMode } = SERVER_CONFIG
+  const fullModeDefault =
+    FULL_SCREEN_INFO !== undefined ? FULL_SCREEN_INFO === 'true' : fullViewMode
+  const { isFullMode } = useGeneral()
+  const { setFullMode } = useGeneralApi()
+
+  useEffect(() => {
+    !isFullMode && gotoPage()
+  }, [])
+
+  return (
+    <Stack overflow="auto">
+      <Stack
+        direction="row"
+        alignItems="center"
+        justifyContent="space-between"
+        gap={1}
+        mx={1}
+        mb={1}
+      >
+        <Stack direction="row">
+          {fullModeDefault && (
+            <SubmitButton
+              data-cy="detail-back"
+              icon={<NavArrowLeft />}
+              tooltip={Tr(T.Back)}
+              isSubmitting={isFetching}
+              onClick={() => unselect()}
+            />
+          )}
+        </Stack>
+
+        <Stack direction="row" alignItems="center" gap={1} mx={1} mb={1}>
+          {!fullModeDefault && (
+            <SubmitButton
+              data-cy="detail-full-mode"
+              icon={isFullMode ? <Collapse /> : <Expand />}
+              tooltip={Tr(T.FullScreen)}
+              isSubmitting={isFetching}
+              onClick={() => {
+                setFullMode(!isFullMode)
+              }}
+            />
+          )}
+          <SubmitButton
+            data-cy="detail-refresh"
+            icon={<RefreshDouble />}
+            tooltip={Tr(T.Refresh)}
+            isSubmitting={isFetching}
+            onClick={() => getComments({ id })}
+          />
+          {typeof unselect === 'function' && (
+            <SubmitButton
+              data-cy="unselect"
+              icon={<Cancel />}
+              tooltip={Tr(T.Close)}
+              onClick={() => unselect()}
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Using Function.prototype.bind and arrows functions as attributes will negatively impact performance in React. Each time the parent is rendered, the function will be re-created and trigger a render of the component causing excessive renders and more memory use.